src/scalarstats.jl

```diff
@@ -524,10 +524,10 @@ function sem(x::AbstractArray, weights::ProbabilityWeights; mean=nothing)
 end
 
 # Median absolute deviation
-@irrational mad_constant 1.4826022185056018 BigFloat("1.482602218505601860547076529360423431326703202590312896536266275245674447622701")
+@irrational mad_to_std_multiplier 1.4826022185056018 BigFloat("1.482602218505601860547076529360423431326703202590312896536266275245674447622701")
```

Comment:
This is type piracy. One must not use `@irrational` outside of Base.Math. The proper way to define irrational constants in packages is to use IrrationalConstants.jl